### PR TITLE
Fix for Makie#614

### PR DESCRIPTION
**Pull request by ffreyer**
This combines the `normal` and `occlusion` buffer to one so that we don't use `Texture{Vec3f0}` anymore. See https://github.com/JuliaPlots/Makie.jl/issues/614

Seems to be working fine with ssao on.
```julia
meshscatter(rand(Point3f0, 100), ssao=true, color=:white, shading=false)
```
Screenshot from 2020-07-17 22-26-34

### CODE REVIEW

**Comment by ffreyer** [comment]
The tests are running super slow for me, but I've gotten past the ssao test without an error.

**Comment by SimonDanisch** [comment]
Yeah, there seems to have a been a major regression in test speeds ... Only thing coming to mind would be the renderloop changes!

**Comment by SimonDanisch** [comment]
Oh wow, completely forgot about this  this pr-.- I just came to the conclusion in https://github.com/JuliaPlots/Makie.jl/issues/671, that we may as well use Vec4f0 - but this is of course more elegant!

File: src/glwindow.jl [hunk 1]
```diff
@@ -35,9 +35,9 @@ mutable struct GLFramebuffer
     objectid::Texture{Vec{2, GLushort}, 2}
     depth::Texture{GLAbstraction.DepthStencil_24_8, 2}
     position::Texture{Vec4f0, 2}
-    normal::Texture{Vec3f0, 2}
+    # NOTE: temporary fix for https://github.com/JuliaPlots/Makie.jl/issues/614
```
Comment (SimonDanisch) [inline src/glwindow.jl:38]: Why temporary? Seems pretty elegant to pack those into one texture ;)

**Comment by SimonDanisch** [comment]
I'd say, this fix is much preferable ;) Any final comments before I merge it? :)

**Comment by SimonDanisch** [comment]
https://www.reddit.com/r/opengl/comments/8fmzk8/redering_into_an_gl_rgba32f_texture_on_intel/

**Comment by SimonDanisch** [comment]
Oh my fix was even erroring -.- I'll just merge this immediately! If you still have any concerns, let me know!

File: src/glwindow.jl (hunk 1 above)
Comment (ffreyer) [reply src/glwindow.jl:38]: I don't remember why I called this temporary... Maybe I was worried that at some point some other post-processing thing will require `normals` but not `occlusion`. Then we'd have to use more vram than necessary or do some weird trickery.

Comment (SimonDanisch) [reply src/glwindow.jl:38]: Well, I think most drivers will pad it to Vec4f0 anyways ;) So, likely no space wasted, if we can't use the fourth component

**Pull request merged**